### PR TITLE
Fix some automation issues found in v1.3.2 release testing

### PR DESCRIPTION
**Pull request by albinsun**
### Which issue(s) this PR fixes:
1. Issue #1464
1. Issue #1463 

### What this PR does / why we need it:
1. `test_1_images.py`
   * Correct wrong image id.
1. `test_5_vm_networks_interact.py`
   * Fix potential problem caused by `unique_name` fixture value is overwritten improperly.
   * Teardown volumes

### Special notes for your reviewer:
n/a

### Additional documentation or context
* Verification (`harvester-runtests/90`)
  image

### CODE REVIEW

**Comment by albinsun** [comment]
Please help review, thx.

File: harvester_e2e_tests/integrations/test_5_vm_networks_interact.py [hunk 1]
```diff
@@ -317,14 +300,19 @@ def test_mgmt_network_connection(
                                             " to VM on management interface"
                                             f" {mgmt_ip} from external host")
 
-        # cleanup VM
-        delete_vm(api_client, unique_name, wait_timeout)
+        # teardown
+        code, data = api_client.vms.get(vm_name)
+        vm_spec = api_client.vms.Spec.from_dict(data)
+        vm_checker.wait_deleted(vm_name)
```
Comment (irishgordo) [inline harvester_e2e_tests/integrations/test_5_vm_networks_interact.py:306]: qu: For all `#teardown` new implementations -> do we want to pass in a wait_timeout for `wait_deleted` [from the virtualmachines.py fixture](https://github.com/harvester/tests/blob/586d6a718573a718f53ee3e7674fb0c647bf373f/harvester_e2e_tests/fixtures/virtualmachines.py#L224-L237) it's keyword arg of `endtime`?
Maybe something like `endtime=wait_timeout+datetime.now()` (pseudo code)

**Review by irishgordo — CHANGES_REQUESTED** [review]
just had a small qu for all new `# teardown` implmentations - otherwise non-blocking, will approve :smile:

**Review by lanfon72 — APPROVED** [review]
LGTM

**Review by irishgordo — APPROVED** [review]

**Pull request merged**